build.gradle

```diff
@@ -295,6 +349,20 @@ dependencies {
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
 
+  contractTestCompile group: 'io.rest-assured', name: 'rest-assured', version: versions.restAssured
+  contractTestCompile group: 'io.rest-assured', name: 'json-path', version: versions.restAssured
+  contractTestCompile group: 'io.rest-assured', name: 'xml-path', version: versions.restAssured
+
+  contractTestCompile group: 'au.com.dius.pact.consumer', name: 'junit5', version: versions.pact
+  contractTestRuntime group: 'au.com.dius.pact.consumer', name: 'junit5', version: versions.pact
+
+  contractTestCompile group: 'au.com.dius.pact.provider', name: 'junit5', version: versions.pact
+  contractTestCompile group: 'au.com.dius.pact.provider', name: 'spring', version: versions.pact
+  contractTestCompile group: 'au.com.dius.pact.provider', name: 'junit5spring', version: versions.pact
+
+  contractTestImplementation group: 'org.projectlombok', name: 'lombok', version: versions.lombok
+  annotationProcessor group: 'org.projectlombok', name: 'lombok', version: versions.lombok
```

Comment:
already imported annotationProcessor for lombok on line 342

Reply:
Indeed. Corrected